src/ngx_http_lua_kong_apple_m1.c

```diff
@@ -39,21 +34,6 @@ typedef struct {
     char **errmsg;
 } ngx_shdict_get_t;
```

Comment:
Let's move these into a separate header file.

Reply:
Also, we should probably rename this struct to something like `ngx_http_lua_kong_shdict_get_t` or something similar. `ngx_shdict_get_t` seems to be a bit odd.